src/components/Person/SinglePerson.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { NavLink } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const SinglePerson: React.FC<Props> = ({ person, people }) => {
+  const { slug } = useParams();
+  const selectedPerson = people.find(identity => slug === identity.slug);
+
+  let motherExists: string | null = person.motherName;
+  let fatherExists: string | null = person.fatherName;
+  const isMotherInList = people.find(
+    identity => identity.name === person.motherName,
+  );
+  const isFatherInList = people.find(
+    identity => identity.name === person.fatherName,
+  );
```

Comment:
Prepare people array in the people page, `Person` component should just get a person and render it, nothing else